app/controllers/concerns/page_tool_tip.rb

```diff
@@ -0,0 +1,22 @@
+module PageToolTip
```

Comment:
Hi @andela-oesho thanks for your PR!

We have encountering many work related for tests inside the project can you provide tests to it?